src/package/nuspec/TestPlatform.Build.nuspec

```diff
@@ -12,6 +12,7 @@
     <iconUrl>http://go.microsoft.com/fwlink/?LinkID=288859</iconUrl>
     <projectUrl>https://github.com/microsoft/vstest/</projectUrl>
     <copyright>© Microsoft Corporation. All rights reserved.</copyright>
+    <tags>vstest visual-studio unittest testplatform mstest microsoft test</tags>
```

Comment:
Searching for packages with `test` shows 1042 packages and `testing` lists 1045. While we're at it, maybe add testing as well. I wish it were easier to find and add tags from nuget ;)

Reply:
Done.